logrus.go

```diff
@@ -82,8 +82,15 @@ const (
 	InfoLevel
 	// DebugLevel level. Usually only enabled when debugging. Very verbose logging.
 	DebugLevel
+	// Not used. It's used to derive the const maxLevel in case
+	// someone needs to enumerate the levels.
+	lastLevel
 )
 
+// This can be used as the highest level possible, if you want to
+// iterate through all log levels, iterate through [0, MaxLevel].
+const MaxLevel Level = lastLevel - 1
```

Comment:
In general this is how I've done it in C programs, so it's obvious, and you don't need to change two lines when adding a new level.